core/tests/test_window.py

```diff
@@ -381,6 +381,49 @@ def test_screen_position(window, app):
     assert window.screen_position == (100, 100)
 
 
+def test_widget_id_reusable_after_window_closes(window, app):
+    """Widget IDs can be reused after the associated widget's window is closed."""
+
+    # Common widget IDs
+    common_widget_ids = {"content": "window_content", "label": "sample_label"}
```

Comment:
I don't see why this is a dictionary containing 2 constants, instead of just using 2 constants. The "dictionary" features of this are completely unused - we just need 2 constant strings.